gis/grass/Portfile

```diff
@@ -71,8 +72,6 @@ configure.args-append \
     --with-pdal=${prefix}/bin/pdal-config \
     --with-tiff-includes=${prefix}/include \
     --with-tiff-libs=${prefix}/lib \
-    --with-png-includes=${prefix}/include \
-    --with-png-libs=${prefix}/lib \
```

Comment:
Do these specific changes remove any PNG functionality in this version of `grass`?

Reply:
No, just changes to configure, (see https://github.com/OSGeo/grass/pull/2679).